bzt/modules/selenium.py

```diff
@@ -39,10 +39,23 @@
     from pyvirtualdisplay import Display
 
 
-class SeleniumExecutor(ScenarioExecutor, WidgetProvider, FileLister):
+class BaseSeleniumExecutor(ScenarioExecutor):
```

Comment:
Maybe have SeleniumExecutor as base class to keep all places that use it untouched?